configuration/log-configuration.yaml

```diff
@@ -76,6 +76,10 @@ options:
     cardano.node.peers.BlockFetchDecision:
       - kind: UserDefinedBK
         name: LiveViewBackend
+  mapScribes:
+    cardano:
+      - JournalSK
```

Comment:
this should be added to 'defaultScribes'

Reply:
> this should be added to 'defaultScribes'

There is that old parsing problem popping up, which I mentioned previously:
```
defaultScribes:
  - - FileSK
    - "node-0.log"
  - - JournalSK
    - "foo"
```
..yields
```
AesonException "Error in $.defaultScribes[1][0]: The key \"JournalSK\" was not found"

cardano-node: Aeson exception:
Error in $.defaultScribes[1][0]: The key "JournalSK" was not found
```